### DISCUSSION TITLE
Nowafpls Module

### DISCUSSION THREAD

**Discussion by Sh4d0wHunt3rX** [blacklanternsecurity/bbot#1406] [General]
That would be great if possible to implement this somehow in BBOT:
https://github.com/assetnote/nowafpls

**Comment by TheTechromancer** [comment]
Whoa neat. @liquidsec

**Comment by liquidsec** [comment]
nowafpls has been amazing. Not exactly sure how we'd use it in bbot? maybe lightfuzz?

**Comment by liquidsec** [comment]
going back through all the discussions, and this is actually a great idea i think we should revisit
